src/pages/workspace/workflows/WorkspaceWorkflowsPage.tsx

```diff
@@ -35,32 +48,42 @@ function WorkspaceWorkflowsPage({policy, route}: WorkspaceWorkflowsPageProps) {
     const ownerPersonalDetails = ReportUtils.getDisplayNamesWithTooltips(OptionsListUtils.getPersonalDetailsForAccountIDs([policy?.ownerAccountID ?? 0], CONST.EMPTY_OBJECT), false);
     const policyOwnerDisplayName = ownerPersonalDetails[0]?.displayName;
     const containerStyle = useMemo(() => [styles.ph8, styles.mhn8, styles.ml11, styles.pv3, styles.pr0, styles.pl4, styles.mr0, styles.widthAuto, styles.mt4], [styles]);
+    const canUseDelayedSubmission = Permissions.canUseWorkflowsDelayedSubmission(betas);
+
+    const onPressAutoReportingFrequency = useCallback(() => Navigation.navigate(ROUTES.WORKSPACE_WORKFLOWS_AUTOREPORTING_FREQUENCY.getRoute(policy?.id ?? '')), [policy?.id]);
 
     const items: ToggleSettingOptionRowProps[] = useMemo(
         () => [
-            {
-                icon: Illustrations.ReceiptEnvelope,
-                title: translate('workflowsPage.delaySubmissionTitle'),
-                subtitle: translate('workflowsPage.delaySubmissionDescription'),
-                onToggle: (isEnabled: boolean) => {
-                    Policy.setWorkspaceAutoReporting(route.params.policyID, isEnabled);
-                },
-                subMenuItems: (
-                    <MenuItem
-                        title={translate('workflowsPage.submissionFrequency')}
-                        titleStyle={styles.textLabelSupportingNormal}
-                        descriptionTextStyle={styles.textNormalThemeText}
-                        // onPress={() => Navigation.navigate(ROUTES.WORKSPACE_WORKFLOWS_AUTOREPORTING_FREQUENCY).getRoute(route.params.policyID))}
-                        // TODO will be done in https://github.com/Expensify/Expensify/issues/368332
-                        description={translate('workflowsPage.weeklyFrequency')}
-                        shouldShowRightIcon
-                        wrapperStyle={containerStyle}
-                        hoverAndPressStyle={[styles.mr0, styles.br2]}
-                    />
-                ),
-                isActive: policy?.harvesting?.enabled ?? false,
-                pendingAction: policy?.pendingFields?.isAutoApprovalEnabled,
-            },
+            ...(canUseDelayedSubmission
+                ? [
+                      {
+                          icon: Illustrations.ReceiptEnvelope,
+                          title: translate('workflowsPage.delaySubmissionTitle'),
+                          subtitle: translate('workflowsPage.delaySubmissionDescription'),
+                          onToggle: (isEnabled: boolean) => {
+                              Policy.setWorkspaceAutoReporting(route.params.policyID, isEnabled);
+                          },
+                          subMenuItems: (
+                              <MenuItem
```

Comment:
This menu item isn't adhering pattern B.
1. Go offline
2. Change submission frequency

Expected: Frequency menu item is greyed out.
Actual: It's not greyed out.

Sidenote: Pattern B works for `Date of month` selector.

https://github.com/Expensify/App/assets/29673073/c5c58b22-4280-4c89-831c-200344f35184

Reply:
It's to be expected since I had to remove `OfflineWithFeedbackComponent` because of the reasons explained [here](https://github.com/Expensify/App/pull/37213#issuecomment-1966663990)

cc @luacmartins

Reply:
hmm weird. I'm really sure, since `OfflineWithFeedbackComponent` should render it [here](https://github.com/Expensify/App/blob/f020194b5a7d4f3d8c8f71f24886b33dedc12066/src/components/OfflineWithFeedback.tsx#L141). We can probably investigate that as part of a follow up